src/omnisharp/OmnisharpManager.ts

```diff
@@ -32,24 +31,31 @@ export class OmnisharpManager {
                 throw new Error('Invalid path specified');
             }
         }
-        //If the path is not a valid path on disk, treat it as a version 
-        return await this.InstallVersionAndReturnLaunchPath(omnisharpPath, useMono, serverUrl, installPath, extensionPath, platformInfo);
+        else if (omnisharpPath == "latest") {
+            return await this.LatestInstallAndReturnLaunchPath(downloader, useMono, serverUrl, versionFilePathInServer, installPath, extensionPath, platformInfo);
+        }
+        //If the path is neither a valid path on disk not the string "latest", treat it as a version 
+        return await this.InstallVersionAndReturnLaunchPath(downloader, omnisharpPath, useMono, serverUrl, installPath, extensionPath, platformInfo);
     }
 
-    public async InstallVersionAndReturnLaunchPath(version: string, useMono: boolean, serverUrl: string, installPath: string, extensionPath: string, platformInfo: PlatformInformation) {
+    public async LatestInstallAndReturnLaunchPath(downloader: OmnisharpDownloader, useMono: boolean, serverUrl: string, versionFilePathInServer: string, installPath: string, extensionPath: string, platformInfo: PlatformInformation) {
+        let version = await downloader.GetLatestVersion(serverUrl, versionFilePathInServer);
+        await downloader.DownloadAndInstallOmnisharp(version, serverUrl, installPath);
```

Comment:
I would expect this to call InstallVersionAndLaunchPath with the version you determined through downloading.